scipy/sparse/linalg/_isolve/tests/test_iterative.py

```diff
@@ -724,6 +737,7 @@ def test_defective_matrix_breakdown(self):
         # The solution should be OK outside null space of A
         assert_allclose(A @ (A @ x), A @ b)
 
+    @pytest.mark.filterwarnings(f"ignore:{CB_TYPE_FILTER}:DeprecationWarning")
```

Comment:
OTOH, I find this line much nicer than (as the failing test tells me to) indenting almost the whole test another level to wrap it in `numpy.testing.suppress_warnings`, or adding it 4x for the individual `gmres` calls.